servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile_inc.glsl

```diff
@@ -20,6 +20,10 @@ layout(push_constant, std430) uniform DrawCall {
 	vec2 uv_offset;
 	uint instance_index;
 	uint pad;
+	uint multimesh_motion_vectors_current_offset;
+	uint multimesh_motion_vectors_previous_offset;
```

Comment:
This needs to be packed like the Forward+ version since we shouldn't use more than 16 bytes on Mobile (except when using the ubershader)

https://github.com/godotengine/godot/blob/7f5c4692923bc7609159b8f4595e9ad321c4c5ab/servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered_inc.glsl#L36-L39